src/libstore/unix/build/derivation-builder.cc

```diff
@@ -2157,7 +2157,7 @@ std::unique_ptr<DerivationBuilder> makeDerivationBuilder(
     }
 
     if (useSandbox)
-        return std::make_unique<LinuxDerivationBuilder>(
+        return std::make_unique<ChrootLinuxDerivationBuilder>(
             store,
             std::move(miscMethods),
             std::move(params));
```

Comment:
I think it would make more sense to move this below?

Reply:
If (can't make suggestion

```
if (params.drvOptions.useUidRange(params.drv))
```
becomes
```
if (!useSandbox && params.drvOptions.useUidRange(params.drv))
```
I think it is fine

Reply:
Done!

Reply:
Err sorry I meant "I think I is mine \[to move the early return below so it is with the other returns]". I.e. this edit was not supposed to be an alternative to the thing I originally requested, but a method how to do it.

Could you make another PR consolidating the returns now?